### DISCUSSION TITLE
Cycles

### DISCUSSION THREAD

**Discussion by GrahamSH-LLK** [osTicket/roadmap#14] [Q&A] [answered]
Hi! I've been following OSTicket 2.0, and I'm really excited. I just saw the new roadmap structure and had a question about which cycle we're in/which issues are in this cycle. Could you post how long cycles are/which cycle we're in? I'm sorry if I've missed something, but I couldn't find anything about this.

**Comment by aydreeihn** [comment, marked as answer]
We recently starting following this model for development, so we're experimenting with 6 week cycles for now. We are currently in a cooldown phase. You can see what we have upcoming in our [project](https://github.com/orgs/osTicket/projects/2).
